src/main/webapp/app/text-assessment/text-assessment.component.ts

```diff
@@ -251,6 +255,31 @@ export class TextAssessmentComponent implements OnInit, OnDestroy, AfterViewInit
             });
         }
     }
+    /**
+     * Load next assessment in the same page.
```

Comment:
It would be good to add more information on the side-effects this function has.
E.g. at the moment it is triggering a route change on an executed api call.